demo/src/main/java/com/midtrans/demo/DemoOrderReviewActivity.java

```diff
@@ -398,7 +398,10 @@ private void initData() {
     }
 
     private CustomerDetails getUserDetail() {
-        return MidtransSDK.getInstance().getTransactionRequest().getCustomerDetails();
+        if (MidtransSDK.getInstance().getTransactionRequest().getCustomerDetails() != null) {
+            return MidtransSDK.getInstance().getTransactionRequest().getCustomerDetails();
+        }
+        return null;
```

Comment:
I think its better to undo the change, since the previous logic also return null

Reply:
Yes you are correct, will revert it